File: androidApp/src/main/java/com/medhelp/medhelp/data/model/analise_price/AnalisePriceList.java
package com.medhelp.medhelp.data.model.analise_price;

import com.google.gson.annotations.SerializedName;

import java.util.ArrayList;
import java.util.List;

public class AnalisePriceList {
    @SerializedName("imgError")
    private boolean error;

    @SerializedName("Message")
    private String message;

    @SerializedName("response")
    private List<AnalisePriceResponse> response = new ArrayList<>();

    public boolean isError() {
        return error;
    }

    public void setError(boolean error) {
        this.error = error;
    }

    public String getMessage() {
        return message;
    }

    public void setMessage(String message) {
        this.message = message;
    }

    public List<AnalisePriceResponse> getResponse() {
        return response;
    }

    public void setResponse(List<AnalisePriceResponse> response) {
        this.response = response;
    }
}

File: shared/src/commonMain/kotlin/com/medhelp/newmedhelp/model/AnalisePriceList.kt
package com.medhelp.newmedhelp.model

import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable

@Serializable
class AnalisePriceList {
    @SerialName("imgError")
    var error = false

    @SerialName("Message")
    var message: String? = null

    @SerialName("response")
    var response: List<AnalisePriceResponse> = ArrayList<AnalisePriceResponse>()
}